README.rst

```diff
@@ -82,6 +64,21 @@ This command will display the external IP address of the EC2 instance.
 
 Run ``python start-aws.py --help`` to see the available options to this command.
 
+Create a staging Docker image
+=============================
```

Comment:
Maybe some of this can be part of start-aws?

Separate what has to be done only once and what is done each time.